src/genophenocorr/analysis/_config.py

```diff
@@ -108,16 +137,20 @@ def pval_correction(self, pval_correction: typing.Optional[str]):
         """
         Set `pval_correction` option.
         """
-        # TODO - check admissible values
-        self._pval_correction = pval_correction
+        if pval_correction in P_VAL_OPTIONS:
```

Comment:
This is wrong. The user should be allow to turn off the p value correction by setting `None`. However, since `None` is not a member of `P_VAL_OPTIONS`, the correction will not be turned off. Instead of that, the user will see a deceiving logging message 
```
Ignoring invalid `pval_correction` value None. Not doing p-value correction.
```
when in fact `bonferroni` will be done.

You must fix this to ensure that only invalid `str`s are complained about and we can turn the correction off.

Reply:
EDIT, actually now I see that `None` in fact IS in `P_VAL_OPTIONS`, so I was not entirely right above.

However, the reporting of what will be done if the user provides an invalid `pval_correction` must still be fixed. The warning message claims that the correction will not be done, but that actually depends on the previous `self._pval_correction` value, which is `bonferroni` by default.

Reply:
Fixed logging message.